### DISCUSSION TITLE
Mi Band WatchFace Editor GUI

### DISCUSSION THREAD

**Discussion by Johnson070** [Johnson070/MiBand-5-watchface-editor#11] [General]
1.0.1.1 (2021-04-19)

- Bug fixes

- Add support MiBand 6(Without building a bin file yet)

- Add simple select images(Hold down the left mouse button to select, and to delete with shift)

- Animation preview(use carefully, the program uses more than 100 MB of RAM)

- (FIXED) If you have a problem with the image, try changing the scale of windows to 100%

- (FIXED) Month/Date add

- (FIXED) Add wind error and today separate temperature

Thank you for downloading the program.

If you find a bug, please write to github or telegram: @vveber

<hr /><em>This discussion was created from the release <a href='https://github.com/Johnson070/MiBand-5-watchface-editor/releases/tag/1.0.1.1'>Mi Band WatchFace Editor GUI</a>.</em>

**Comment by kimsun102** [comment]
I've been really waiting for this! But when will you update to building a bin file? I really want to make watch faces. Anyway, thank you so much for make this program! (I'm not a english speaker, so please ignore my bad english...)

+I think you should to edit "Add support MiBand 5(Without building a bin file yet)" to "Add support MiBand 6(Without building a bin file yet)"

**Reply by Johnson070** [reply]
)))

**Reply by Johnson070** [reply]
The next update will be in a week or two.  It all depends on the developer MiBand6WF.  I partially help him in development

**Reply by kimsun102** [reply]
Thanks a lot!

**Reply by Johnson070** [reply]
In main directory I put test watchfaces

**Reply by Johnson070** [reply]
https://github.com/Johnson070/MiBand-5-watchface-editor/tree/main/MiBand6JsonWatchfaces

**Comment by kimsun102** [comment]
How did you make it? Did you make it by using the program(Mi Band WatchFace
Editor)?

2021년 4월 20일 (화) 오후 4:48, johnson070 ***@***.***>님이 작성:

**Comment by kimsun102** [comment]
Then can you tell me how to building a .bin file?(I'm sorry to ask so many
questions.)

2021년 4월 20일 (화) 오후 4:48, johnson070 ***@***.***>님이 작성:

**Reply by Johnson070** [reply]
bin file assembly is temporarily not supported, in the next update

**Comment by kimsun102** [comment]
Ok, thanks. 😁

2021년 4월 20일 (화) 오후 5:24, johnson070 ***@***.***>님이 작성:

**Reply by Johnson070** [reply]
Check new update, add support compile bin file)

**Comment by kimsun102** [comment]
Thanks a million!

2021년 4월 22일 (목) 오전 9:07, johnson070 ***@***.***>님이 작성:

**Comment by ldicdican** [comment]
How to install the custom watch face for band 6?
Files on "Android/data/com.xiaomi.wearable/files/WatchFace " does not have bin files

**Comment by ldicdican** [comment]
![Screenshot_20210422-224913_ES File Explorer](https://user-images.githubusercontent.com/83022771/115735398-07cdb400-a3bd-11eb-9e74-594b53425d9e.jpg)
